packages/dds/sequence/api-report/sequence.api.md

```diff
@@ -65,6 +65,14 @@ export function appendIntervalPropertyChangedToRevertibles(interval: SequenceInt
 // @internal
 export function appendSharedStringDeltaToRevertibles(string: SharedString, delta: SequenceDeltaEvent, revertibles: SharedStringRevertible[]): void;
 
+// @alpha (undocumented)
```

Comment:
FYI--we should be mindful here on current state of the 2.0 API tagging. It looks like SharedString & dependents have already been tagged as alpha and aren't still internal, so it makes sense to flag this as alpha as well. But keep an eye out for this while you're merging the PR.

Reply:
Let's also be sure to document any new API members we're adding. Ideally none of these new items should have the `// (undocumented)` comments.

Reply:
Makes sense, I only added the alpha tags when I got api-extractor errors - is the guidance still to add them when required by the compiler?